website/docs/r/windows_web_app_slot.html.markdown

```diff
@@ -0,0 +1,625 @@
+
+* `app_service_name` - (Required) The name of the Windows Web App this Deployment Slot will be part of.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Windows Web App Slot should exist. Changing this forces a new Windows Web App Slot to be created.
+
+* `service_plan_id` - (Required) The ID of the Service Plan that this Windows App Service will be created in.
+
+* `site_config` - (Required) A `site_config` block as defined below.
+
+---
+
+* `app_settings` - (Optional) A map of key-value pairs of App Settings.
+
+* `auth_settings` - (Optional) A `auth_settings` block as defined below.
+
+* `backup` - (Optional) A `backup` block as defined below.
+
+* `client_affinity_enabled` - (Optional) Should Client Affinity be enabled?
+
+* `client_certificate_enabled` - (Optional) Should Client Certificates be enabled?
+
+* `client_certificate_mode` - (Optional) The Client Certificate mode. Possible values include `Optional` and `Required`. This property has no effect when `client_cert_enabled` is `false`
+
+* `connection_string` - (Optional) One or more `connection_string` blocks as defined below.
+
+* `enabled` - (Optional) Should the Windows Web App Slot be enabled? Defaults to `true`.
+
+* `https_only` - (Optional) Should the Windows Web App Slot require HTTPS connections.
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `key_vault_reference_identity_id` - (Optional) The User Assigned Identity ID used for accessing KeyVault secrets. The identity must be assigned to the application. [For more information see - Access vaults with a user-assigned identity](https://docs.microsoft.com/en-us/azure/app-service/app-service-key-vault-references#access-vaults-with-a-user-assigned-identity)
+
+* `logs` - (Optional) A `logs` block as defined below.
+
+* `storage_account` - (Optional) One or more `storage_account` blocks as defined below.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Windows Web App Slot.
+
+---
+
+A `action` block supports the following:
+
+* `action_type` - (Required) Predefined action to be taken to an Auto Heal trigger. Possible values include: `Recycle`.
+
+* `custom_action` - (Optional) A `custom_action` block as defined below.
+
+* `minimum_process_execution_time` - (Optional) The minimum amount of time in `hh:mm:ss` the Windows Web App Slot must have been running before the defined action will be run in the event of a trigger.
+
+---
+
+A `active_directory` block supports the following:
+
+* `client_id` - (Required) The ID of the Client to use to authenticate with Azure Active Directory.
+
+* `allowed_audiences` - (Optional) Specifies a list of Allowed audience values to consider when validating JWTs issued by Azure Active Directory.
+
+~> **Note:** The `client_id` value is always considered an allowed audience.
```

Comment:
(.. and must be included within it?)

Reply:
No, it's implicit, so should probably always be omitted. I'll update to that effect since it's likely to cause a diff if the service stops returning it at any point.